openhands/runtime/base.py

```diff
@@ -132,6 +132,8 @@ def __init__(
 
         self.user_id = user_id
 
+        self.prev_token: SecretStr | None = None
```

Comment:
can you add a TODO to get rid of this once we're done debugging? it should be reasonably safe but I don't like holding onto secrets